src/main/java/org/codehaus/groovy/transform/trait/TraitASTTransformation.java

```diff
@@ -498,7 +500,7 @@ private void processField(final FieldNode field, final MethodNode initializer, f
                         ClassNode.EMPTY_ARRAY,
                         returnS(initCode.getExpression())
                 );
-                helper.addMethod(fieldInitializer);
+                ClassNodeUtils.addGeneratedMethod(helper, fieldInitializer);
```

Comment:
I am unsure whether this one is needed. The whole helper class is marked as generated. Does the coverage tool not recognize that?

Reply:
You are right, whole helper class is marked as generated and this is unnecessary. Reverted it with new commit.